2/debian9/2.190/Dockerfile

```diff
@@ -65,13 +65,17 @@ EXPOSE 50000
 
 ENV COPY_REFERENCE_FILE_LOG $JENKINS_HOME/copy_reference_file.log
 
-USER ${user}
-
 COPY jenkins-support /usr/local/bin/jenkins-support
 COPY jenkins.sh /usr/local/bin/jenkins.sh
-ENTRYPOINT ["/bin/tini", "--", "/usr/local/bin/jenkins.sh"]
-
 # from a derived Dockerfile, can use `RUN plugins.sh active.txt` to setup /usr/share/jenkins/ref/plugins from a support bundle
 COPY plugins.sh /usr/local/bin/plugins.sh
 COPY install-plugins.sh /usr/local/bin/install-plugins.sh
 COPY monitoring-plugin.groovy /usr/share/jenkins/ref/init.groovy.d/monitoring-plugin.groovy
+
+RUN chown ${user}:${group} /usr/local/bin/*.sh \
```

Comment:
Why not setting the owner above?

COPY --chown=${user} plugins.sh /usr/local/bin/plugins.sh

Reply:
+1 👍 . Fixed.

Reply:
It didn't work. I rolled back the commit and sent you sponge link in pvt.  :(